src/LegacyMSBuildResolver/LegacyResolver.txt

```diff
@@ -0,0 +1,7 @@
+# -------------------------------------------------------------------------------
+# use a completely new error number and keep messages in their surrounding groups
+# -------------------------------------------------------------------------------
+assemblyResolutionFoundByAssemblyFoldersKey,"Found by AssemblyFolders registry key"
```

Comment:
@KevinRansom Is there a chance these numbers could clash with other FCS ones? How these numbers are going to be accessed via `FSharpDiagnostic`?